module-system/module-implementations/sov-evm/src/query.rs

```diff
@@ -15,14 +15,21 @@ use crate::Evm;
 
 #[rpc_gen(client, server, namespace = "eth")]
 impl<C: sov_modules_api::Context> Evm<C> {
-    // TODO https://github.com/Sovereign-Labs/sovereign-sdk/issues/502
```

Comment:
Let's add a check to the integration for this.  Here is an example of how we do it for another endpoint:
https://github.com/Sovereign-Labs/sovereign-sdk/blob/nightly/examples/demo-rollup/tests/evm/mod.rs#L132
&
https://github.com/Sovereign-Labs/sovereign-sdk/blob/nightly/examples/demo-rollup/tests/evm/mod.rs#L201